doc/INSTALL-ubuntu.md

```diff
@@ -0,0 +1,49 @@
+# Installation on Ubuntu
+
+Tested on Lubuntu Eoan (19.04) using `bash` as the shell.
+
+1. Install dependencies:
+
+        sudo apt install --install-suggests libsdl-gfx1.2-dev
+        sudo apt install qt4-default coreutils build-essential
+
+2. Install BASS:
+   - Visit <http://www.un4seen.com/> and click "Download"
+     next to the penguin at the top of the page
+   - Unzip the downloaded bass24-linux into a directory of your choice.
+     **Note:** Create the directory and cd into it before unzipping, since the
+     ZIP does not have a directory name.  E.g.,
+
+         $ mkdir bass24 && cd bass24 && unzip ../bass24-linux.zip
```

Comment:
Shouldn't `unzip -d bass24 bass24-linux.zip` do the trick here?

This is what we do on CI: https://github.com/rocket/rocket/blob/master/.travis.yml#L25

Reply:
Nice!  I actually haven't used `-d` before :) .  Will fix.